packages/itwin/reports-config-widget/src/widget/components/ReportsContainer.scss

```diff
@@ -5,9 +5,6 @@
 @import "~@itwin/itwinui-css/scss/variables";
 
 .reports-container {
-  position: absolute;
-  right: 0;
-  left: 0;
```

Comment:
You're going to have to bump up the minimum iTwin.js version to ^3.3.5. Removing this workaround will break the styling for any version below 3.3.5 as they don't include this fix: https://github.com/iTwin/itwinjs-core/pull/4266

Reply:
Change was reverted to avoid the problems that package upgrades might bring.